Makefile

```diff
@@ -1,12 +1,16 @@
 CALICO_DIR=$(shell git rev-parse --show-toplevel)
 VERSIONS_FILE?=$(CALICO_DIR)/_data/versions.yml
+IMAGES_FILE=
 JEKYLL_VERSION=pages
 HP_VERSION=v0.2
 DEV?=false
 CONFIG=--config _config.yml
 ifeq ($(DEV),true)
 	CONFIG:=$(CONFIG),_config_dev.yml
 endif
+ifneq ($(IMAGES_FILE),)
+	CONFIG:=$(CONFIG),/config_images.yml
```

Comment:
Is this correct? I can't understand what's going on here

Reply:
Essentially, if an IMAGES_FILE is provided, then we'll do two things:
- Update the config argument to include `/config_images.yml` (This part)
- Mount the provided IMAGES_FILE into the container as `/config_images.yml`. (in `make _site`)